### ISSUE TITLE
RuntimeError: This event loop is already running

### ISSUE THREAD

**Issue by fritol** [akaszynski/keepa#68]
```
----> 3 api = keepa.Keepa(accesskey)
      4 #/graphimage?key=<yourAccessKey>&domain=<domainId>&asin=<ASIN>

D:\anaconda3\envs\amazonmws\lib\site-packages\keepa\interface.py in __init__(self, accesskey)
   2133     def __init__(self, accesskey):
   2134         """ Initializes object """
-> 2135         self.__dict__["parent"] = run_and_get(AsyncKeepa.create(accesskey))
   2136 
   2137     def __setattr__(self, attr, value):

D:\anaconda3\envs\amazonmws\lib\site-packages\keepa\interface.py in run_and_get(coro)
   2087         loop = asyncio.new_event_loop()
   2088     task = loop.create_task(coro)
-> 2089     loop.run_until_complete(task)
   2090     return task.result()
   2091 

D:\anaconda3\envs\amazonmws\lib\asyncio\base_events.py in run_until_complete(self, future)
    590         """
    591         self._check_closed()
--> 592         self._check_running()
    593 
    594         new_task = not futures.isfuture(future)

D:\anaconda3\envs\amazonmws\lib\asyncio\base_events.py in _check_running(self)
    550     def _check_running(self):
    551         if self.is_running():
--> 552             raise RuntimeError('This event loop is already running')
    553         if events._get_running_loop() is not None:
    554             raise RuntimeError(

RuntimeError: This event loop is already running
```

**Comment by akaszynski** [comment]
As noted in #65, have you installed the `asyncio` library?

**Comment by LeSonicDev** [comment]
If you read code of run_and_get you will find it starts the event loop. Here is how I call keepa api:

api = await keepa.AsyncKeepa.create('your token goes here')
...
batch_cat_by_ids = await api.category_lookup(batch_cat_ids_str)
...

**Comment by akaszynski** [comment]
Should be fixed in `keepa==`1.3.0`.
